### ISSUE TITLE
qt-full 5.12 - permission denied

### ISSUE THREAD

**Issue by sjau** [NixOS/nixpkgs#65620]
**Describe the bug**

Can't upgrade my system because of qt full producing errors:

```
created 648 symlinks in user environment
/nix/store/01xalm9xmzv24s6hwr4l6pp71cz67rsx-stdenv-linux/setup: line 1289: /nix/store/9gf9zs5351hpb8wrx05nfrn4d8dg8ylq-qt-full-5.12.0/bin/qt.conf: Permission denied
building '/nix/store/zpcn75gpis4gdc7ijhasanhrg5xmbwwp-reload-container.drv'...
building '/nix/store/0avqnqj2ixc59by406nlh7hxdr6h40ak-sddm.pam.drv'...
building '/nix/store/3lzcsjvpmllk67sz7zh9s2h49h8an4kh-set-environment.drv'...
builder for '/nix/store/my7knnjbfyx30km6lclaglwm1nhgiy76-qt-full-5.12.0.drv' failed with exit code 1
building '/nix/store/kqd8llgkbjc3zcx2jwb9iwhc9dzc3132-slim.pam.drv'...
building '/nix/store/0gy6kfx9akw7jfajm298l65mgnzfba2x-smartd-notify.sh.drv'...
cannot build derivation '/nix/store/y05mmqdaljp98a66709kjk6jzwiihi6s-system-path.drv': 1 dependencies couldn't be built
building '/nix/store/8xiwb5ipnsb2gn6q1m1zmig5ain3bq2c-systemd-user.pam.drv'...
building '/nix/store/10xm5zi70nlhs13jrh37rsd1l6hizcrk-torrc.drv'...
building '/nix/store/0pmkyp7finqwrzwccdxf5b95w7j3lys9-udev-path.drv'...
building '/nix/store/y4j4yzc3kdfclsvbi61hwm59h4lxvspf-unit-alsa-store.service.drv'...
building '/nix/store/gn2ghyhswyrzlc7f9spm3bwap0wnq9fv-unit-audit.service.drv'...
building '/nix/store/bziy7faaf5ia6zxndjs9mzgw9qwn7x97-unit-avahi-daemon.service.drv'...
cannot build derivation '/nix/store/80dws8nc3vqqqvnp0f96q8h8s8vn5bgn-nixos-system-subi-19.09pre186857.239fffc90d7.drv': 1 dependencies couldn't be built
error: build of '/nix/store/80dws8nc3vqqqvnp0f96q8h8s8vn5bgn-nixos-system-subi-19.09pre186857.239fffc90d7.drv' failed
```

**Metadata**

 - system: `"x86_64-linux"`
 - host os: `Linux 5.1.11, NixOS, 19.09.git.83ba5af (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(hyper): `"nixos-19.09pre185402.5eac339829b"`
 - channels(root): `"nixos-19.09pre186857.239fffc90d7"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

**Comment by worldofpeace** [comment]
cc @ttuegel

**Comment by ttuegel** [comment]
Fixed in #65446.
